platform/ios/MGLAccountManager.m

```diff
@@ -51,8 +51,9 @@ + (void) setAccessToken:(NSString *) accessToken {
     [[MGLAccountManager sharedManager] setAccessToken:accessToken];
 
     // Update MGLMapboxEvents
-    // NOTE: This is (likely) the initial setup of MGLMapboxEvents
-    [MGLMapboxEvents sharedManager];
+    // NOTE: This is (likely) the initial setup of MGLMapboxEvents (via implicit call to +[MGLMapboxEvents sharedManager])
+    // Turn the Mapbox Turnstile to Count App Users
+    [MGLMapboxEvents pushEvent:MGLEventTypeAppUserTurnstile withAttributes:nil];
```

Comment:
GTK about the evolving different ways into the SDK.  I just moved it to `-[MGLMapboxEvents init]` and it works great.  Thanks!